market/utils/approval-decision-utils/meta.json

```diff
@@ -0,0 +1,22 @@
+{
+	"$schema": "https://json-schema.axonivy.com/market/10.0.0/meta.json",
+	"id": "approval-decision-utils",
+	"name": "Approval Decision Utils",
+	"description": "Empower applications with Approval Decision: Accelerate decision-making and streamline processes",
```

Comment:
Standardize decision-making in business processes to meet regulatory, compliance, and transparency requirements.

Reply:
hi @andreasbalsiger, I've modified the artifact description. Kindly review the PR, thank you!